class.jetpack-connection-banner.php

```diff
@@ -166,7 +166,13 @@ function enqueue_banner_scripts() {
 	 *
 	 * @since 4.4.0
 	 */
-	function render_banner() { ?>
+	function render_banner() {
+		$disconnection_notice = Jetpack_Options::get_option( 'disconnection_notice', array() );
+		$notice = sprintf( __( 'Your connection to WordPress.com broken please try reconnecting!', 'jetpack' ) );
```

Comment:
should be "... to WordPress.com is broken..."

Reply:
I'm no wordsmith, but this could sound friendlier and better aligned with the "Set Up Jetpack" call to action when first connecting. Perhaps along the lines of:

> Jetpack has lost its connection to WordPress.com. Please set up to continue enjoying all of Jetpack's features.

Will this text have a "Set Up Jetpack" button next to it?

cc: @Automattic/jetpack-design

Reply:
How about:

Whoops -- Jetpack's connection to WordPress.com is broken. Please reconnect Jetpack to get everything up and running again.

Shorter:

Jetpack's connection to WordPress.com is broken! Please reconnect to get everything up and running again.

(In this context, when they've already started working with Jetpack, "set up" feels a little off to me, and the "enjoy JP's features" is a little more salesy than I think we need to be with people who've already chosen to try JP.